src/Form/atoms/SelectField.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useCallback, useState, SelectHTMLAttributes } from 'react';
+import styled, { css } from 'styled-components';
+import Label, { StyledLabel } from './Label';
+import { ISelectSizes } from '../index';
+import Icon, { IconWrapper } from '../../Icons/Icon';
+
+
+const Container = styled.div<{ styleSize?: ISelectSizes, }>`
+
+${({ styleSize }) => (styleSize === 'small') && css`
+  ${StyledLabel} {
+      span {
+        margin-bottom: 6px;
+      }
+    }
+  `}
+`;
+
+const StyledSelect = styled.select<{ styleSize?: ISelectSizes, activePlaceholder: boolean, selectWidth?: string }>`
+  box-sizing: border-box;
+  outline: none;
+  border-radius: 3px;
+  height: 40px;
+  padding: 0 25px 0 12px;
+  appearance: none;
+  width: ${({ selectWidth }) => selectWidth ? selectWidth : '220px'};
+
+  &::-ms-expand {
+    display: none;
+  }
+
+  ${({ theme }) => css`
+    border: 1px solid ${theme.styles.form.input.default.normal.borderColor};
+    font-family: ${theme.fontFamily.data};
+    ${theme.typography.form.input.value.normal};
+  `};
+
+  ${({ theme: { typography }, styleSize }) => (styleSize === 'small') && css`
+    height: 30px;
+    padding: 0 25px 0 5px;
+    ${typography.form.input.value.compact};
+  `};
+
+  ${({ theme, activePlaceholder }) => activePlaceholder && css`
+    ${theme.typography.form.input.placeholder.normal};
+  `};
+
+  ${({ theme, styleSize, activePlaceholder }) => (styleSize === 'small') && activePlaceholder && css`
+    ${theme.typography.form.input.placeholder.compact};
+  `};
+
+  option {
+    font-style: normal;
+    ${({ theme }) => css`
+      ${theme.typography.form.input.value.normal};
+    `};
+
+    ${({ theme: { typography }, styleSize }) => (styleSize === 'small') && css`
+      ${typography.form.input.value.compact};
+    `};
+  }
+`;
+
+const SelectWrapper = styled.div`
+  position: relative;
+  display: inline-block;
+
+  ${IconWrapper} {
+    position: absolute;
+    top: calc(50% - 7px);
+    right: 10px;
+  }
+`;
+
+interface ILabel {
+  htmlFor: string
+  labelText: string
+}
```

Comment:
I don't think this is consistent with the other form elements we have. This seems like a sensible choice although we need to consider consistency first. What are your thoughts here?

Reply:
I wanted to make having a Label as optional in this component but if label is used I wanted to make htmlFor a required to enforce the developer to provide that value.

We could drop the Label functionality if you want and just have the Select component.

Reply:
Perhaps something like this might be a good middle ground. That way the label is option but if you do provide it then both of those props become mandatory.

```
interface ILabel {
  htmlFor: string
  text: string
}

interface IProps {
    label?: ILabel
}
```

Reply:
I have updated as suggested and added sameRow prop optional to make the label in the same row that the select.